assets/css/translate.scss

```
@@ -0,0 +1,40 @@
+// Translation results of " Section x" in articles
+// Autor: Olsza
+// License: CC-BY-4.0
+
+// Enter the locale code and the translation of "Section" in the language.
+// The country code should be the same as the value of the lang attribute in html (use "-" instead of "_")
+$section_translation: (
+        "de": "",
+        "el": "",
+        "en": "section",
```

```suggestion
        "en": "Section",
```

Shouldn't this be capitalized?

---

Thx